rhel/SPECS/dbaas-tools.spec

```diff
@@ -0,0 +1,64 @@
+%define debug_package %{nil}
+
+%global project_aws         kubernetes-sigs
+%global repo_aws            aws-iam-authenticator
+%global provider_prefix_aws github.com/%{project_aws}/%{repo_aws}
+%global commit_aws          d7c0b2e9131faabb2b09dd804a35ee03822f8447
+%global shortcommit_aws     %(c=%{commit_aws}; echo ${c:0:7})
+
+%define full_pmm_version 2.0.0
+%global install_golang 1
+
+%define build_timestamp %(date -u +"%y%m%d%H%M")
+%define release         1
+%define rpm_release     %{release}.%{build_timestamp}%{?dist}
+
+Name:           dbaas-tools
+# the line below is sed'ed by build-server-rpm script to set a correct version
```

Comment:
It shouldn't – that version has nothing to do with PMM version

Reply:
Ok, I'll remove it